src/dotnet/commands/dotnet-restore/Program.cs

```diff
@@ -24,16 +24,12 @@ public static int Run(string[] args)
                 Description = "Restores dependencies listed in project.json"
             };
 
-            // Parse --quiet, because we have to handle that specially since NuGet3 has a different
-            // "--verbosity" switch that goes BEFORE the command
```

Comment:
If NuGet uses `--verbosity` BEFORE the "restore" command, does passing in `--verbosity` into `dotnet restore` work?

Ex. `dotnet restore --verbosity Verbose`.  Does that actually use print out verbose messages?